__tests__/integration.test.js

```diff
@@ -193,4 +194,69 @@ describe("/api/articles/:article_id/comments", () => {
     })
   })
 
+  test("POST: Successfull posts to this endpoint should respond with 201 and the newly added comment inluding comment_id, author, body, votes, created_at, article_id", () => {
+    return request(app)
+    .post("/api/articles/3/comments")
+    .send({
+      username: "theRealDumbledore",
+      body: "It does not do to dwell on dreams, and forget to live"
+    })
+    .expect(201)
+    .then(({body})=>{
+      const {comment} = body
+      expect(comment).toEqual({
+        comment_id: 19,
+        article_id: 3,
+        author: "theRealDumbledore",
```

Comment:
These tests have absolutely made my day, so thank you! :D Amazing! _Obviously..._